stock_move_consolidated/wizard/stock_move_history_view.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_stock_move_history" model="ir.ui.view">
+        <field name="name">Stock Move Report at Date</field>
+        <field name="model">stock.move.consolidation</field>
+        <field name="arch" type="xml">
+            <form string="Choose your date">
+                <group>
+                    <group>
+                        <field name="date_from"/>
+                    </group>
+                    <group>
+                        <field name="date_to"/>
+                    </group>
+                </group>
+
+                <footer>
+                    <button name="stock_move_till_date" string="View" type="object" class="btn-primary"/>
+                    <button string="Cancel" class="btn-secondary" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record id="consolidate_stock_move_action" model="ir.actions.act_window">
```

Comment:
we need access to this action without having to mark any products.

Right now i only get the action button after having marked some products, whereas we want this action to simply be based on date

Reply:
okay, then the button on the list view will work. I just added both, but I will remove it